### ISSUE TITLE
Using ssf "arguments" in config.json don't work

### ISSUE THREAD

**Issue by gregor-hh** [securesocketfunneling/ssf#43]
Hello again ;)
I'am trying to start ssfc -c config.json without additional parameters. I put the needed ssfs arguments in the json config but it don't work.

Maybe I doing something wrong or it's a bug? 

my config.json:

{
  "ssf": {
	**"arguments"; "-L 8081:127.0.0.1:5080 xxx.domain.net -p 443",**
    "tls" : {
      "ca_cert_path": "./certs/trusted/ca.crt",
      "cert_path": "./certs/certificate.crt",
      "key_path": "./certs/private.key",
      "key_password": "",
      "dh_path": "./certs/dh4096.pem",
      "cipher_alg": "DHE-RSA-AES256-GCM-SHA384"
    },
    "http_proxy" : {
      "host": "",
      "port": "",
      "credentials": {
        "username": "",
        "password": "",
        "domain": "",
        "reuse_ntlm": "true",
        "reuse_nego": "true"
      }
    },
    "services": {
      "datagram_forwarder": { "enable": true },
      "datagram_listener": {
        "enable": true,
        "gateway_ports": false
      },
      "stream_forwarder": { "enable": true },
      "stream_listener": {
        "enable": true,
        "gateway_ports": false
      },
      "file_copy": { "enable": false },
      "shell": {
        "enable": false,
        "path": "C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\powershell.exe",
        "args": "-File -"
      },
      "socks": { "enable": true }
    }
  }
}

after starting "ssfc.exe -c config.json" , the following error is displayed:
`[2017-04-06 12:06:43][critical] command line: parsing failed: the option '--host' is required but missing`
`[2017-04-06 12:06:43][error] client: wrong command line arguments
`

Any ideas? Syntax issue? 

Thank You for help.
BR Gregor

**Comment by securesocketfunneling** [comment]
It's not a bug, it's a feature ;)

Did you build your executables from the sources or you are using those from the web site?
The `arguments` configuration is pretty new and is not present in __2.2__.
It will appear in __2.3__.

You have to refer to [this README](https://github.com/securesocketfunneling/ssf/tree/2.2.0) for 2.2 documentation.

By the way, there is a typo in your `config.json` file:
```
"arguments"; "-L 8081:127.0.0.1:5080 xxx.domain.net -p 443",
```

should be 
```
"arguments": "-L 8081:127.0.0.1:5080 xxx.domain.net -p 443",
```

**Comment by gregor-hh** [comment]
Thank You very much, I used the Webdownload ;-) Ok than thats the reason. The typo I saw. 

Ok, well then i must wait ;-)

**Comment by securesocketfunneling** [comment]
Hello @gregor-hh,

We released [SSF 3.0.0](https://github.com/securesocketfunneling/ssf/releases/tag/3.0.0) and the `arguments` config key is now supported in the pre-built binaries.
Check it out and do not hesitate to send us feedback.
